.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   test:
-    name: Lint and test
+    name: Lint, build and test
```

Comment:
let's keep the name for consistency (it's called the same across all our repos, regardless of what exactly this step does)

Reply:
Okay 👍

Reply:
Updated